uawidgets/new_node_dialogs.py

```diff
@@ -54,6 +61,8 @@ def __init__(self, parent, title, server):
 
     def _store_state(self):
         self.settings.setValue("last_namespace", self.nsComboBox.currentIndex())
+        self.settings.setValue("last_auto_nodeid_state", self.nodeidCheckBox.isChecked())
```

Comment:
I will change the name. It defaults to auto id enabled (nodeid field hidden). If user changes it, it's just saved instead of resetting every time you add another node.